packages/select/src/ResourceSelect.js

```diff
@@ -25,6 +25,7 @@ const ResourceSelect = ({
   waitUntilFocused,
   defaultToOnlyOption,
   shouldSearch,
+  additionalPostGetArgs,
   ...rest
 }) => {
```

Comment:
We also are going to need to modify the `cacheUniq` logic so if the props change the select knows to bust the cache and refetch the resource if something from `additionalPostGetArgs` updates